servers/visual/visual_server_raster.h

```diff
@@ -68,27 +69,29 @@ class VisualServerRaster : public VisualServer {
 	List<FrameDrawnCallbacks> frame_drawn_callbacks;
 
 	void _draw_margins();
-	static void _changes_changed() {}
+	static void _changes_changed(int p_priority) {
+		if (p_priority) {
+			;
+		}
+	}
 
 public:
-	//if editor is redrawing when it shouldn't, enable this and put a breakpoint in _changes_changed()
-	//#define DEBUG_CHANGES
```

Comment:
Ah yes there is an existing comment here just below. I can add another one above though if it will make more clear.